ci/tests/general/general_vdisk.py

```diff
@@ -65,16 +65,23 @@ def create_volume(size, vpool, name=None, loop_device=None, root_client=None, wa
                                                         vdisk_name=name if name is not None else uuid.uuid4())
         if root_client is None:
             root_client = SSHClient('127.0.0.1', username='root')
-        root_client.run('truncate -s {0}G {1}'.format(size, location))
 
-        if loop_device is not None:
-            root_client.run('losetup /dev/{0} {1}'.format(loop_device, location))
-            root_client.dir_create('/mnt/{0}'.format(loop_device))
-            root_client.run('parted /dev/{0} mklabel gpt'.format(loop_device))
-            root_client.run('parted -a optimal /dev/{0} mkpart primary ext4 0% 100%'.format(loop_device))
-            root_client.run('partprobe')
-            root_client.run('mkfs.ext4 /dev/{0}'.format(loop_device))
-            root_client.run('mount -t ext4 /dev/{0} /mnt/{0}'.format(loop_device))
+        try:
+            if loop_device is not None:
+                root_client.run('truncate -s {0}G {1}'.format(size, location))
+                root_client.run('losetup /dev/{0} {1}'.format(loop_device, location))
+                root_client.dir_create('/mnt/{0}'.format(loop_device))
+                root_client.run('parted /dev/{0} mklabel gpt'.format(loop_device))
+                root_client.run('parted -a optimal /dev/{0} mkpart primary ext4 0% 100%'.format(loop_device))
+                root_client.run('partprobe | true')
+                root_client.run('mkfs.ext4 /dev/{0}'.format(loop_device))
+                root_client.run('mount -t ext4 /dev/{0} /mnt/{0}'.format(loop_device))
+        except Exception as ex:
+            cmd = """
+                umount /mnt/{0};
+                losetup -d /dev/{0};
+                rm {1}""".format(loop_device, location)
+            print root_client.run(cmd)
```

Comment:
If the ```create_volume``` calls fails (for some reason, e.g. the system can for some reason not truncate a new file), the test used to fail, but will now silently continue. Won't that cause issues by subsequent code which assumes the disk was created?